addons/arsenal/script_component.hpp

```diff
@@ -14,4 +14,21 @@
     #define DEBUG_SETTINGS DEBUG_SETTINGS_ARSENAL
 #endif
 
+#ifdef DISABLE_COMPILE_CACHE
+    #define RECOMPILE
+#endif
+
 #include "\z\ace\addons\main\script_macros.hpp"
+
+
+// if use just GVAR here, "Undefined variable" error occurs
```

Comment:
Where?

Reply:
instead of `missionNamespace getVariable QGVAR()`. Looks like there is changed namespace in BIS code

Reply:
Ah yeah. BIS code runs in uiNamespace for whatever reason.